system/webserver/webserverd.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+from flask import Flask, render_template, Response, request
+from system.webserver.helpers import *
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello_world():
+  return render_template("index.html")
```

Comment:
```suggestion
```
Can we remove this?

Reply:
no clue if it'll break something if we do

Reply:
Let's change the name to `index()`.